sentry-samples/kmp-app-mvvm-di/README.md

```diff
@@ -0,0 +1,32 @@
+# Sentry KMP Demo App
```

Comment:
@buenaflor what are the steps for compiling and running on Android and iOS? Which IDE? I can't run on Xcode nor AS right now.

Reply:
>  SDK location not found. Define a valid SDK location with an ANDROID_HOME environment variable or by setting the sdk.dir path in your project's local properties file at '/Users/marandaneto/Github/sentry-kotlin-multiplatform/local.properties'.

Mention to export `ANDROID_HOME`

Reply:
Mention `:sentry-samples:kmp-app:androidApp:assembleDebug` would compile the Android version